business/istio_status.go

```diff
@@ -50,8 +50,9 @@ func (ics *IstioComponentStatus) merge(cs IstioComponentStatus) IstioComponentSt
 
 const (
 	Healthy     string = "Healthy"
-	Unhealthy   string = "Unhealthy"
 	NotFound    string = "NotFound"
+	NotReady   	string = "NotReady"
```

Comment:
Here it seems there is a minor space/tab issue:
![image](https://user-images.githubusercontent.com/1662329/110808945-c35edc80-8284-11eb-8e8b-2e6cc80ef589.png)